README.md

```diff
@@ -1,5 +1,22 @@
 # HTML&CSS judge for [Dodona](https://dodona.ugent.be/)
 
+> The HTML judge has two ways to evaluate a student's submitted solution. In *comparison mode*, the structure of the submitted solution is compared to a model solution based on generic comparison criteria. The teacher can set how strictly that comparison should be done.
+>
+> *Checklist mode* offers even more possibilities because the submitted solution is checked against the criteria explicitly set up for the exercise by the teacher. The students get to see as feedback which criteria have or have not been met.  
```

Comment:
```suggestion
> *Checklist mode* offers even more possibilities because the submitted solution is checked against the criteria explicitly set up for the exercise by the teacher. The feedback that students get to see shows which criteria have or have not been met.  
```